va_explorer/templates/va_data_management/show.html

```diff
@@ -24,7 +24,9 @@
 <div class="tab-content">
   <div class="tab-pane fade show active" id="record" role="tabpanel" aria-labelledby="record-tab">
     <div class="row mt-3">
-      <a class="btn btn-primary" href="{% url 'data_management:edit' id=id %}">Edit Record</a>
+      {% if perms.va_data_management.change_verbalautopsy %}
```

Comment:
@smacadam Also need to wrap the two buttons at the bottom of the page, i.e.: 

`{% if perms.va_data_management.change_verbalautopsy %} 
          <a class="btn btn-primary mx-2" href="{% url 'data_management:reset' id=id %}">Reset to Original</a></p>
          <a class="btn btn-primary mx-2" href="{% url 'data_management:revert_latest' id=id %}">Revert Most Recent Change</a>
{% endif %} `

Reply:
Added check